### PR TITLE
Data.py with data cleaning functions, updated requirements.txt

### PR DESCRIPTION
**Pull request by sneaky-hobbitses**
Data.py with data cleaning functions, updated requirements.txt

### CODE REVIEW

File: data_scientist_skills/data.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+def get_data(df1_dir, df2_dir): #Relative path to each data file
+    #Get data
+    df1 = pd.read_csv(df1_dir)
+    df2 = pd.read_csv(df2_dir)
+    #Drop Columns
+    df1.drop(columns='Unnamed: 0', inplace=True)
+    df2.drop(columns = ['Unnamed: 0', 'index'], inplace = True)
+    #Concat to full dataframe
+    df = pd.concat([df1, df2], ignore_index=True, axis=0)
```
Comment (honnsey) [inline data_scientist_skills/data.py:13]: Once we have the merged dataframe, we can drop a few more columns as discussed? 

df.drop(columns = ['Revenue', 'Competitors', 'Easy Apply'], inplace= True)

'Easy Apply' refers to whether you can directly apply from Glassdoor I think, so doesn't help us very much

File: data_scientist_skills/data.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    return df
+
+def get_description(df):
+    return pd.DataFrame(df['Job Description'])
+
+def clean(description):   # .apply to description or title column
+    stop_words = set(stopwords.words('english'))
```
Comment (honnsey) [inline data_scientist_skills/data.py:24]: maybe add a doc string here, for good practice?
''' Function returns cleaned text from one input string, applicable for description columns and/or title column'''

File: data_scientist_skills/data.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+def get_data(df1_dir, df2_dir): #Relative path to each data file
+    #Get data
+    df1 = pd.read_csv(df1_dir)
+    df2 = pd.read_csv(df2_dir)
+    #Drop Columns
+    df1.drop(columns='Unnamed: 0', inplace=True)
+    df2.drop(columns = ['Unnamed: 0', 'index'], inplace = True)
+    #Concat to full dataframe
+    df = pd.concat([df1, df2], ignore_index=True, axis=0)
+
```
Comment (RemoteSalad) [inline data_scientist_skills/data.py]: I wanted to add one more line to rename the column names to a more standard format:

`df.columns = [column.replace(' ', '_').lower() for column in df.columns]`

File: data_scientist_skills/data.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+from nltk.tokenize import word_tokenize
+from nltk.corpus import stopwords
+from nltk.stem import WordNetLemmatizer
+import string
+import unidecode
+
+def get_data(df1_dir, df2_dir): #Relative path to each data file
```
Comment (RemoteSalad) [inline data_scientist_skills/data.py]: Can we assume that we can find the data in the raw_data directory? i.e. no need to pass parameters

File: data_scientist_skills/data.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    # Lemmatize
+    lemmatized_description = [WordNetLemmatizer().lemmatize(word, pos="v") # V for Verb
+                            for word in description_tokenized]
+    cleaned_sentence = ' '.join(word for word in lemmatized_description)
+    return cleaned_sentence
+
+
+def get_cleaned_description(dataframe):
+    df = get_description(dataframe)
+    df['Cleaned Description'] = df['Job Description'].apply(clean)
```
Comment (RemoteSalad) [inline data_scientist_skills/data.py:48]: needs the updated column names

File: data_scientist_skills/data.py [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    df2 = pd.read_csv("data_scientist_skills/raw_data/DataScientist.csv")
+    #Concat to full dataframe
+    df = pd.concat([df1, df2], ignore_index=True, axis=0)
+    #Drop columns and reformat names
+    df.drop(columns=['Unnamed: 0', 'index', 'Revenue', 'Competitors', 'Easy Apply'], inplace=True)
+    df.columns = [column.replace(' ', '_').lower() for column in df.columns]
+    return df
+
+def get_description(df):
+    return pd.DataFrame(df['Job Description'])
```
Comment (RemoteSalad) [inline data_scientist_skills/data.py:20]: needs the updated column names

**Pull request merged**